CRM/Contribute/BAO/Contribution.php

```diff
@@ -4684,6 +4695,7 @@ public static function completeOrder(&$input, &$ids, $objects, $transaction, $re
       civicrm_api3('Contribution', 'sendconfirmation', array(
         'id' => $contribution->id,
         'payment_processor_id' => $paymentProcessorId,
+        'is_email_receipt' => (array_key_exists('is_email_receipt', $values) ? $values['is_email_receipt'] : 0),
```

Comment:
@agileware Instead of using ternary operator, can we use existing function which does this ?

     'is_email_receipt' =>  CRM_Utils_Array::value('is_email_receipt', $values);